ui/about_window.cpp

```diff
@@ -29,25 +29,72 @@
 #include <sstream>
 
 #include "common/dive_version.h"
+#include "utils/version_info.h"
 
 // =================================================================================================
 // AboutDialog
 // =================================================================================================
 
 AboutDialog::AboutDialog(QWidget *parent)
 {
-    // Build version string
+    m_main_layout = new QVBoxLayout;
+    CreateHeaderLayout();
+    CreateVersionLayout();
+    CreateLicenseLayout();
+    CreateButtonLayout();
+
+    // Disable help icon, set size, title, and layout
+    setWindowFlags(windowFlags() & ~Qt::WindowContextHelpButtonHint);
+    setMinimumSize(640, 480);
+    setWindowTitle("About Dive");
+    setLayout(m_main_layout);
+}
+
+void AboutDialog::CreateHeaderLayout()
+{
+    m_icon = new QIcon(":/images/dive.ico");
+    m_icon_label = new QLabel();
+    m_icon_label->setPixmap(m_icon->pixmap(64, 64));
+    m_icon_label->setFixedSize(64, 64);
+
     std::ostringstream os;
-    os << DIVE_PRODUCT_NAME << std::endl;
+    os << DIVE_PRODUCT_NAME << " " << Dive::GetVersionNumberString() << std::endl;
+    os << std::endl;
     os << DIVE_PRODUCT_DESCRIPTION << std::endl;
-    os << DIVE_COPYRIGHT_DESCRIPTION << std::endl;
     os << std::endl;
-    os << "Version ";
-    os << DIVE_VERSION_MAJOR << "." << DIVE_VERSION_MINOR << "." << DIVE_VERSION_REVISION;
+    os << DIVE_COPYRIGHT_DESCRIPTION << std::endl;
     m_build_information = new QLabel(os.str().c_str());
     m_build_information->setWordWrap(true);
 
-    // Load third party license information
+    m_header_layout = new QHBoxLayout;
+    m_header_layout->addWidget(m_icon_label);
+    m_header_layout->addWidget(m_build_information);
+
+    m_main_layout->addLayout(m_header_layout);
+}
+
+void AboutDialog::CreateVersionLayout()
+{
+    m_version_label = new QLabel("Version details:");
+
+    std::ostringstream os;
+    os << Dive::GetVersionDetailedSummary(DIVE_INSTALL_DIR_PATH, DIVE_BUILD_TYPE);
+    m_version_details = new QPlainTextEdit();
+    m_version_details->setPlainText(os.str().c_str());
```

Comment:
GetVersionDetailedSummary returns a string, shouldn't need to use std::ostringstream
```suggestion
    m_version_details = new QPlainTextEdit();
    m_version_details->setPlainText(Dive::GetVersionDetailedSummary(DIVE_INSTALL_DIR_PATH, DIVE_BUILD_TYPE).c_str());
```